### ISSUE TITLE
ACICreate failing

### ISSUE THREAD

**Issue by brian-doss** [dgkanatsios/AzureContainerInstancesManagement#15]
Hello, I'm new to this project and I've recently deployed this to my Azure tenant in order to test/configure. I was following the demo notes and was trying to run ACICreate for an image we have in our Azure Container Registry. However, I'm currently receiving the following error below.

ERROR: Error: The image 'myregistry.azurecr.io/image_name:latest' in container group 'image_name' is not accessible. Please check the image and registry credential.

You mention to follow the instructions [here](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-using-azure-container-registry), however I appear to be doing something incorrectly currently. Can you elaborate further on how to use this project with an ACR?

Thank you,

Brian

**Comment by dgkanatsios** [comment]
Did you upload your container image to Azure Container Registry? If so, you should replace 'myregistry.azurecr.io/image_name:latest' with the relevant properties of your image.

**Comment by brian-doss** [comment]
Yes, the container image is uploaded to the ACR. I edited that error line to exclude our ACR/image details.
On the instructions [here](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-using-azure-container-registry), the only thing that seemed relevant to my need was adding the following JSON:

"imageRegistryCredentials": [
  {
    "server": "imageRegistryLoginServer",
    "username": "imageRegistryUsername",
    "password": "imageRegistryPassword"
  }
]



I'm unsure if this is the correct items I need to add and also where to add this. I had added this to the 'function.json' file in the ACICreate function in order to test, but that came back with the error I've posted.

Thank you,

Brian
